lib/core/src/Cardano/Wallet/Network/Light.hs

```diff
@@ -227,43 +204,43 @@ mkBlockSummary light old new = BlockSummary
     , query = getAddressTxs light old new
     }
 
-blockHeightToInteger :: Quantity "block" Word32 -> Integer
-blockHeightToInteger (Quantity n) = fromIntegral n
-
 {-------------------------------------------------------------------------------
     Logging
 -------------------------------------------------------------------------------}
 data LightLayerLog
-    = MsgLightRollForwardStable ChainPoint BlockHeader BlockHeader BlockHeader
-    | MsgLightRollForwardUnstable ChainPoint BlockHeader BlockHeader
-    | MsgLightRollBackward ChainPoint ChainPoint
+    = MsgLightRollForward
+        ChainPoint BlockHeader BlockHeader BlockHeader
+    | MsgLightRolledForward BlockHeader
+    | MsgLightRollBackward
+        ChainPoint ChainPoint
     deriving (Show, Eq, Generic)
 
 instance ToText LightLayerLog where
     toText = \case
-        MsgLightRollForwardStable cp_ from_ to_ tip -> T.unwords
-            [ "LightLayer roll forward (stable region):"
-            , "chain_point: ", toText $ show cp_
-            , "from: ", toText $ show from_
-            , "to: ", toText $ show to_
-            , "tip: ", toText $ show tip
-            ]
-        MsgLightRollForwardUnstable from_ to_ tip -> T.unwords
-            [ "LightLayer roll forward (unstable region):"
-            , "from: ", toText $ show from_
-            , "to: ", toText $ show to_
-            , "tip: ", toText $ show tip
-            ]
-        MsgLightRollBackward from_ to_ -> T.unwords
-            [ "LightLayer roll backward:"
-            , "from: ", toText $ show from_
-            , "to: ", toText $ show to_
-            ]
+        MsgLightRollForward cp_ from_ to_ tip ->
+            T.unwords
+                [ "LightLayer started rolling forward:"
+                , "chain_point: ", pretty cp_
+                , "from: ", pretty from_
+                , "to: ", pretty to_
+                , "tip: ", pretty tip
+                ]
+        MsgLightRolledForward bh ->
+            T.unwords
+                [ "LightLayer finished rolling forward:"
+                , "last block: ", pretty bh
+                ]
+        MsgLightRollBackward from_ to_ ->
+            T.unwords
+                [ "LightLayer roll backward:"
+                , "from: ", pretty from_
+                , "to: ", pretty to_
+                ]
 
 instance HasPrivacyAnnotation LightLayerLog
 
 instance HasSeverityAnnotation LightLayerLog where
     getSeverityAnnotation = \case
-        MsgLightRollForwardStable{} -> Debug
-        MsgLightRollForwardUnstable{} -> Debug
-        MsgLightRollBackward{} -> Debug
+        MsgLightRollForward{} -> Notice
+        MsgLightRolledForward{} -> Notice
+        MsgLightRollBackward{} -> Notice
```

Comment:
`Notice` is for debugging? 🤔🤪 We may want to remove this in the final version — put it into tech debt so that we don't forget about it?

Reply:
Changed to Debug.